onnxruntime/core/optimizer/qdq_transformer/selectors_actions/shared/utils.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+#include "core/graph/basic_types.h"
+#include "core/providers/nnapi/nnapi_builtin/nnapi_lib/NeuralNetworksTypes.h"
+#include "../qdq_selectors.h"
+#include "../../../selectors_actions/helpers.h"
+
+namespace onnxruntime {
+
+class GraphViewer;
+class Node;
+
+namespace QDQ {
+
+struct Selector {
+  using OpVersionsMap = std::unordered_map<std::string, std::vector<ONNX_NAMESPACE::OperatorSetVersion>>;
+
```

Comment:
If we'll generally be looking up a specific operator version, an unordered_map may be a better fit than a vector.

Reply:
thought operatorsetversion would be a set of ints.. Can you clarify how would this unordered_map be used/how does the map key/value look like?